### PR TITLE
Add Travis build configuration

### PR DESCRIPTION
**Pull request by flibbertigibbet**
Also removes unnecessary log statement that gets thrown out a lot during test runs.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+sudo: false # use new container-based infrastructure, to enable caching on public repo
+language: java
+jdk: oraclejdk7
+before_script: /bin/true
+install: /bin/true # skip install step (runs gradle assemble by default)
+script: mvn -B -q clean package # run in quiet batch mode (to not overwhelm the Travis log)
```
Comment (CloudNiner) [inline .travis.yml:6]: Assuming maven is installed already?

**Comment by CloudNiner** [comment]
+1 on travis success!

File: .travis.yml (hunk 1 above)
Comment (flibbertigibbet) [inline .travis.yml:6]: Yes.

**Comment by flibbertigibbet** [comment]
Passed on my fork.  Apparently the maven git plugin barfed because another PR got merged in; I'll see if there's an easy fix for that.

**Comment by flibbertigibbet** [comment]
I'm going to try closing this PR and opening a new one to see if that fixes it.

**Pull request closed without merging**